Find a functionally equivalent code:android.view.Window.setWindowAnimations:COMMENT
Method Modifier: public      
Comment:/**
 * Specify custom animations to use for the window, as per
 * {@link WindowManager.LayoutParams#windowAnimations
 * WindowManager.LayoutParams.windowAnimations}.  Providing anything besides
 * 0 here will override the animations the window would
 * normally retrieve from its theme.
 */

@@ -1,7 +1,5 @@
 {
     final WindowManager.LayoutParams attrs = getAttributes();
     attrs.windowAnimations = resId;
-    if (mCallback != null) {
-        mCallback.onWindowAttributesChanged(attrs);
-    }
+    dispatchWindowAttributesChanged(attrs);
 }

